Automattic/wp-calypso
# Fix/jetpack cloud backup status styles no backup yet

#### Changes proposed in this Pull Request
**Asana:** 1142395350490785-as-1173751161125817

Update the "No backups are available yet" view to match the design. Add the correct icon.

Before:
![image](https://user-images.githubusercontent.com/9832440/82047875-77b2c500-96ab-11ea-9924-1aa8cae3ad56.png)

![image](https://user-images.githubusercontent.com/9832440/82047835-65d12200-96ab-11ea-8828-dd6a39c669df.png)


After:
![image](https://user-images.githubusercontent.com/9832440/82047530-dd528180-96aa-11ea-85ac-315ae1803d66.png)

![image](https://user-images.githubusercontent.com/9832440/82047549-e6435300-96aa-11ea-805b-e42dc6adc973.png)

*Depend on this PR: https://github.com/Automattic/wp-calypso/pull/42256

#### Testing instructions

- Apply the changes
- Check if match with the design and you can see the correct icon.
